packages/cursorless-vscode-core/commands.ts

```diff
@@ -0,0 +1,11 @@
+import * as vscode from "vscode";
+
+export const showDocumentation = () =>
+  vscode.env.openExternal(vscode.Uri.parse("https://www.cursorless.org/docs/"));
+
+export const showQuickPick = () =>
+  vscode.commands.executeCommand("workbench.action.quickOpen", ">Cursorless");
+
+export const commandIds = {
+  showQuickPick: "cursorless.showQuickPick",
+};
```

Comment:
we could prob use this more widely; needed it for this one because the status bar refers to this command id so didn't want to duplicate it